src/util/shmem/fd_shmem_private.h

```diff
@@ -136,8 +137,18 @@ static inline char *                         /* ==buf always */
 fd_shmem_private_path( char const * name,    /* Valid name */
                        ulong        page_sz, /* Valid page size (normal, huge, gigantic) */
                        char *       buf ) {  /* Non-NULL with FD_SHMEM_PRIVATE_PATH_BUF_MAX bytes */
-  return fd_cstr_printf( buf, FD_SHMEM_PRIVATE_PATH_BUF_MAX, NULL, "%s/.%s/%s",
+                       
+  buf = fd_cstr_printf( buf, FD_SHMEM_PRIVATE_PATH_BUF_MAX, NULL, "%s/.%s/%s",
                          fd_shmem_private_base, fd_shmem_page_sz_to_cstr( page_sz ), name );
+  if ( getenv( "FD_FFI" ) ) {
+    char tmp[FD_SHMEM_PRIVATE_PATH_BUF_MAX];
+    char prefix[9] = "/mnt/.fd";
+    memcpy( tmp, prefix, 8 );
+    memcpy( tmp + 8, buf, FD_SHMEM_PRIVATE_PATH_BUF_MAX - 8 );
+    memset( buf, 0, FD_SHMEM_PRIVATE_PATH_BUF_MAX );
+    memcpy( buf, tmp, FD_SHMEM_PRIVATE_PATH_BUF_MAX );
+  }
+  return buf;
```

Comment:
This is a dumb hack I put in because wksp doesn't look in the right place when calling from Rust -- should properly fix

Reply:
yah just remove all this I guess